build.gradle

```diff
@@ -430,9 +430,14 @@ ext {
                     name   : 'hystrix-core',
                     modules: ['hystrix-serialization']
             ],
+            graalSubstrate             : [
```

Comment:
why do we need `graalSubstrate`?. It does not seem to be used.

Reply:
To not break existing applications